mkt/extensions/models.py

```diff
@@ -149,6 +146,20 @@ def __unicode__(self):
         return u'%s: %s' % (self.pk, self.name)
 
 
+# Maintain ElasticSearch index.
+@receiver(models.signals.post_save, sender=Extension,
+          dispatch_uid='extension_index')
+def update_search_index(sender, instance, **kw):
+    instance.get_indexer().index_ids([instance.id])
+
+
+# Delete from ElasticSearch index on delete.
+@receiver(models.signals.post_delete, sender=Extension,
+          dispatch_uid='extension_unindex')
+def delete_search_index(sender, instance, **kw):
+    instance.get_indexer().unindex(instance.id)
```

Comment:
Are we removing from the index or just filtering them out? For apps we filter so things like the lookup tool can still find them.